src/content/developers/tutorials/hello-world-smart-contract-fullstack/index.md

```diff
@@ -0,0 +1,1548 @@
+
+### Step 1: Clone the starter files {#step-2-clone-the-starter-files}
+
+First, go to the [hello-world-part-four github repository](https://github.com/alchemyplatform/hello-world-part-four-tutorial) to get the starter files for this project. Clone this repository into your local environment.
+
+When you open this cloned `hello-world-part-four` repository, you'll notice that it contains two folders: `starter-files` and `completed`.
+
+- `starter-files` contains the starter files \(essentially the React UI\) for this project. In this tutorial, **we will be working in this directory**, as you learn how to bring this UI to life by connecting it to your Ethereum wallet and the Hello World smart contract that you published on Etherscan in [Part 3](https://docs.alchemy.com/alchemy/tutorials/hello-world-smart-contract/submitting-your-smart-contract-to-etherscan).
+- `completed` contains the entire completed tutorial and is there for you as a **reference** **if you get stuck.**
+
+Next, open your copy of `starter-files` to your favorite code editor, and then navigate into your `src` folder.
+
+All of the code we'll write will live under the `src` folder. We'll be editing the `HelloWorld.js` component and the `util/interact.js` javascript files to give our project Web3 functionality.
+
+### Step 2: Check out the starter files {#step-2-check-out-the-starter-files}
+
+Before we start coding, it's super important that we figure out what's already provided for us in the starter files.
+
+#### Get your react project running {#get-your-react-project-running}
+
+Let's start by running the React project in our browser. The beauty of React is that once we have our project running in our browser, any changes we save will be updated live in our browser.
+
+To get the project running, navigate to the root directory of the `starter-files` folder, and the run `npm install` in your terminal to install the dependencies of the project:
+
+```bash
+cd starter-files
+npm install
+```
+
+Once those have finished installing, run `npm start` in your terminal:
+
+```bash
+npm start
+```
+
+Doing so should open [http://localhost:3000/](http://localhost:3000/) in your browser, where you'll see the frontend for our project. It should consist of one field \(a place to update the message stored in your smart contract\), a "Connect Wallet" button, and an "Update" button.
+
+If you try clicking "Connect Wallet" or "Update" buttons, you'll notice that they don't work—that's because we still need to program their functionality! :\)
```

Comment:
```suggestion
If you try clicking either button, you'll notice that they don't work—that's because we still need to program their functionality.
```